frontend/components/Form/TextArea.vue

```diff
@@ -63,10 +63,12 @@
     },
     maxLength: {
       type: Number,
+      default: -1,
```

Comment:
Not sure if -1 is the best default.

Reply:
I'm not entirely sure on this one either, either `-1` or maybe `Number.MAX_VALUE` from Typescript?

Reply:
@katosdev thoughts?